committing.rst

```diff
@@ -242,8 +242,7 @@ When it is determined that a pull request needs to be backported into one or
 more of the maintenance branches, a core developer can apply the labels
 ``needs backport to X.Y`` to the pull request.
 
-After the pull request has been merged, it can be backported using
-cherry_picker.py_.
+After the pull request has been merged, miss-islington (bot) will first try to do the backport automatically. In case that miss-islington is unable to do it, then the pull request author or the core developer who merged it should look into backporting it themselves, it can be backported using cherry_picker.py_.
```

Comment:
I'd tweak this last phrase to be "... backporting it themselves, using the backport generated by `cherry_picker.py` as a starting point."

We know the cherry_picker.py output will only be a starting point in these cases, as the cases where it applies cleanly are also the ones where miss-islington is able to generate a PR automatically.